index.html

```diff
@@ -1 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 
+    
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <header class="toolbar">
+      <h3 id="home">Home</h3>
```

Comment:
- [ ] Since `Home` is a link to the homepage of your website please use an anchor tag(a) instead of `h3`.